### PR TITLE
Use upstream Makefiles and scripts instead of bazel

### PR DESCRIPTION
**Pull request by inteon**

### CODE REVIEW

**Comment by irbekrm** [comment]
> Use upstream Makefiles and scripts instead of bazel

Can you please link where they come from?

File: prow/Makefile
Comment (inteon) [reply prow/Makefile:24]: I assume this is handy when you want to test a config change without using git?
Do I add a comment or do I remove these targets?

File: prow/pj-on-kind.sh
Comment (irbekrm) [inline prow/pj-on-kind.sh:22]: I don't think we want to remove this comment or add a reference to `prow.k8s.io` which is not what we are targeting

File: prow/pj-on-kind.sh
Comment (irbekrm) [inline prow/pj-on-kind.sh:31]: I would rather keep this file checked into the codebase than pull in a random script from upstream

Comment (irbekrm) [reply prow/pj-on-kind.sh:31]: FWIW we had a discussion about this already https://github.com/jetstack/testing/pull/526#discussion_r667956641

File: prow/Makefile
Comment (irbekrm) [reply prow/Makefile:24]: tbh, it is going to be inconsistent in either way. My main concern is that folks will think that this is how config changes should be applied. A warning that this is just for local testing and any applied changes should be either reset or pushed to Git shoudl be ok 👍🏼
 and perhaps in future we can make it so that the way Prow and Prow config gets applied is more consistent instead of one from dev laptop, the other from Git

File: prow/cluster/labelsync_cronjob.yaml
Comment (irbekrm) [inline prow/cluster/labelsync_cronjob.yaml:31]: There is probably a huge diff here as the last time labelsync was pushed to our registry was in 2019 https://console.cloud.google.com/gcr/images/jetstack-build-infra/eu/prow-labelsync?authuser=1&project=jetstack-build-infra
Please take a look at the release notes and add a note whether there is any diff in how it works that would affect us.

File: prow/cluster/mem-limit-range_limitrange.yaml
Comment (irbekrm) [inline prow/cluster/mem-limit-range_limitrange.yaml:2]: Why do we remove this?

Comment (inteon) [reply prow/cluster/mem-limit-range_limitrange.yaml:2]: Because it is yaml for the worker cluster/ is not deployed on the prow cluster. Maybe I should place it in a separate dir?

Comment (irbekrm) [reply prow/cluster/mem-limit-range_limitrange.yaml:2]: No, that's my bad - and I see that the workers cluster anyway does not have any limitrange in test-pods namespace, so should be fine to remove

File: prow/cluster/labelsync_cronjob.yaml
Comment (inteon) [reply prow/cluster/labelsync_cronjob.yaml:31]: @irbekrm where do I find these release notes? Do I manually check the git diff?

Comment (irbekrm) [reply prow/cluster/labelsync_cronjob.yaml:31]: I normally just skim through Prow announcements, I see that this is now moved to https://docs.prow.k8s.io/docs/announcements/

File: prow/Makefile
Comment (inteon) [reply prow/Makefile]: done

**Review by irbekrm — APPROVED** [review]
Thank you for making these improvements! Adding an lgtm with a hold for you to merge when ready

/lgtm
/hold

**Comment by inteon** [comment]
I applied the changes, and it seems to be working still.
/unhold

**Pull request merged**
